ci/ci.sh

```diff
@@ -181,9 +181,9 @@ test_python() {
     bazel test --config=ci \
       --build_tests_only $(./ci/run/bazel_export_options) \
       --test_env=PYTHONPATH="${PYTHONPATH-}${pathsep}${WORKSPACE_DIR}/python/ray/pickle5_files" \
+      --test_env=CI="1" \
+      --test_env=RAY_CI_POST_WHEEL_TESTS="1" \
       --test_env=USERPROFILE="${USERPROFILE}" \
-      --test_env=CI=1 \
-      --test_env=RAY_CI_POST_WHEEL_TESTS=1 \
```

Comment:
This seems to have solved a problem where the quoted `USERPROFILE` value somehow blocked setting `CI=1` and `RAY_CI_POST_WHEEL_TESTS=1`

Reply:
Interesting, not sure what's going on there but glad it worked!